test/functional/apps/saved_objects_management/edit_saved_object.ts

```diff
@@ -99,5 +121,48 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
       const objects = await PageObjects.settings.getSavedObjectsInTable();
       expect(objects.includes('A Dashboard')).to.be(false);
     });
+
+    it('preserves the object references when saving', async () => {
+      const testVisualizationUrl =
+        '/management/kibana/objects/savedVisualizations/75c3e060-1e7c-11e9-8488-65449e65d0ed';
+
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaSavedObjects();
+
+      let objects = await PageObjects.settings.getSavedObjectsInTable();
+      expect(objects.includes('A Pie')).to.be(true);
```

Comment:
These steps don't seem necessary for this test?

Reply:
This was mostly to insert webdriver implicit timing to wait for the listing page.

I will replace this with just `await PageObjects.settings.getSavedObjectsInTable()`